src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Shape randomShape = Shape.values()[random.nextInt(Shape.values().length)];
+        // This requirement looks strange, but as you wish :-)
+        // ColorSupplier with public String getRandomColor() method - for generating random string,
```

Comment:
Thanks for the comments,
but it is better to remove them

Reply:
Remove. Thanks!